api/controllers/userController.js

```diff
@@ -0,0 +1,25 @@
+const { userService } = require('../services')
+const {catchAsync} = require('../utils/error')
+
+const signUp = catchAsync (async(req, res) => {
+    try {
+        const { name, email, password, phoneNumber, birthday} = req.body;
+        if ( !name || !email || !password || !phoneNumber || !birthday ) {
+            const error = new Error('KEY_ERROR')
+            error.statusCode = 400
+
+            throw error
+        }
+        const insertId = await userService.signUp(name, email, password, phoneNumber, birthday)
+        return res.status(201).json({insertId});
+    } catch (error) {
+        console.error(error) 
+		
+		return res.status(error.statusCode).json({ message: error.message });
+    }
+
+})
```

Comment:
1. controller function을 catchAsync 함수에 argument로 넣어줘서 사용하고 있기 때문에 controller를 포함한  service, model에는 try catch가 작성되지 않아도 됩니다.
2. 코드 가독성을 위해 줄바꿈을 적절히 활용해주세요! 
3. 마찬가지로 indentation이 과하거나 라인별로 안맞는 부분이 있는데 일괄적으로 수정 부탁드리겠습니다~!
```suggestion
const signUp = catchAsync (async(req, res) => {
    const { name, email, password, phoneNumber, birthday} = req.body;
        
    if ( !name || !email || !password || !phoneNumber || !birthday ) {
        const error = new Error('KEY_ERROR')
        error.statusCode = 400

        throw error
    }
        
    const insertId = await userService.signUp(name, email, password, phoneNumber, birthday)
        
    return res.status(201).json({insertId});	
})
```